packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -646,7 +646,13 @@ class TextSelectionOverlay {
     required Offset globalGesturePosition,
     required TextPosition currentTextPosition,
   }) {
-    final Offset globalRenderEditableTopLeft = renderEditable.localToGlobal(Offset.zero);
+    final RenderBox? overlay = Overlay.of(context, rootOverlay: true).context.findRenderObject() as RenderBox?;
```

Comment:
This class probably indirectly stores a reference to the target Overlay somewhere, so you don't have to find it again?

Reply:
I just looked and couldn't find one, only references to the OverlayEntries.